validation/lagom-validation/user-management.md

```diff
@@ -0,0 +1,109 @@
+# User Validation
+Every invalid equivalence class returns an error. Every valid class returns an empty sequence.
+
+## Address
+
+### Street (String)
+*validCharactersStreet* := Letters ∪ Umlauts ∪ Digits ∪ Whitespace ∪ {ß , .  -}
+- vStreet1: street.length ∈ [1, 50] AND street has only valid characters
+- iStreet1: street.length ∉ [1, 50] 
+- iStreet2: street contains characters not defined in *validCharactersStreet*
+
+### houseNumber (String)
+Valid house numbers are of the form "42", "42b", and "53a-c". Letters may be lower- or uppercase.
```

Comment:
do you trim spaces, i.e. is "42 b" valid?

Reply:
No, we do not.